<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadFileDialog</class>
 <widget class="QDialog" name="LoadFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализ и выравнивание долей</string>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Анализ аудио...</string>
     </property>
     <property name="styleSheet">
      <string>font-size: 14px; font-weight: bold; color: white;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="styleSheet">
      <string>QProgressBar {
    border: 2px solid palette(mid);
    border-radius: 5px;
    text-align: center;
    background-color: palette(base);
    color: white;
}
QProgressBar::chunk {
    background-color: #4CAF50;
    border-radius: 3px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>font-size: 12px; color: white; padding: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="preliminaryBPMLabel">
     <property name="text">
      <string>Предварительный BPM (на основе Mixxx): -- BPM</string>
     </property>
     <property name="styleSheet">
      <string>font-size: 11px; color: white; padding: 2px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="computedBPMLabel">
     <property name="text">
      <string>Вычисленный BPM (уверенность --%): -- BPM</string>
     </property>
     <property name="styleSheet">
      <string>font-size: 11px; color: white; padding: 2px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="deviationLabel">
     <property name="text">
      <string>Среднее отклонение: --%</string>
     </property>
     <property name="styleSheet">
      <string>font-size: 11px; color: white; padding: 2px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="markIrregularCheckbox">
     <property name="text">
      <string>Пометить неровные доли</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>font-size: 12px; color: white; padding: 5px;</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="fixButton">
       <property name="text">
        <string>Выровнять</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #4CAF50;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton:disabled {
    background-color: #666666;
    color: #cccccc;
}</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skipButton">
       <property name="text">
        <string>Пропустить</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string>QPushButton {
    background-color: #f44336;
    color: white;
    border: none;
    padding: 8px 16px;
    border-radius: 4px;
    font-size: 12px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #da190b;
}
QPushButton:disabled {
    background-color: #666666;
    color: #cccccc;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
